packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.1]
+
+### Fixed
+
+- fix: add guard if `KeyringController:withKeyring` fails when called in `NotificationServicesController` ([#5514](https://github.com/MetaMask/core/pull/5514))
```

Comment:
Nit: do you think we can remove the fix here ? As this is already under Fixed.

Reply:
Ah good catch, I'll do this!

Reply:
Updated https://github.com/MetaMask/core/pull/5528/commits/12b59fbc7f45c7c49b0eba463d67ed57b9946dbc